csharp-mode.el

```diff
@@ -8,6 +8,7 @@
 ;; Version    : 0.11.0
 ;; Keywords   : c# languages oop mode
 ;; X-URL      : https://github.com/emacs-csharp/csharp-mode
+;; Package-Requires: ((emacs "26.1"))
```

Comment:
I think all package should have this. So I add one.

Reply:
This is fine to me, however, I'm not sure we have to go for 26.1. I think we can go for 25.1, since that is what `tree-sitter` supports

Reply:
Or did you chose 26.1 for a reason? I know tree-sitter-indent goes for 26.1, but I don't think it has to

Reply:
Fixed! ;)